### PR TITLE
DOC-1415: Identify existing REST API Examples in RS docs and simplify the presentation

### PR DESCRIPTION
**Pull request by cmilesb**
Staging Link: https://docs.redis.com/staging/jira-doc-1415/rs/

Part of the RS REST API Cleanup. Changed any REST API examples in /rs/ from cURL examples to "VERB https://[host][:port]/v1/[endpoint]".

Did not touch any calls in the user security page, per Lance.

### CODE REVIEW

**Review by rrelledge — CHANGES_REQUESTED** [review]
This looks good so far. I left a couple of initial suggestions. One more thing you might consider is adding some links to the relevant REST API request pages.

File: content/rs/administering/cluster-operations/removing-node.md [hunk 1]
```diff
@@ -82,12 +82,13 @@ To remove a node using the admin console:
 1. Once the process finishes, the node is no longer shown in
     the UI.
 
-To remove a node using the REST API, use the `/v1/nodes/3/actions/remove` endpoint with the JSON data and the "Content-Type: application/json" header.
+To remove a node using the REST API, use the [`/v1/nodes/node_id/actions/remove`]({{< relref "/rs/references/rest-api/requests/nodes/actions#post-node-action" >}}) endpoint with the JSON data and the "Content-Type: application/json" header.
```
Comment (rrelledge) [inline content/rs/administering/cluster-operations/removing-node.md:85]:
```suggestion
To remove a node using the REST API, use the [`/v1/nodes/<node_id>/actions/remove`]({{< relref "/rs/references/rest-api/requests/nodes/actions#post-node-action" >}}) endpoint with the JSON data and the "Content-Type: application/json" header.
```

File: content/rs/administering/access-control/password-rotation.md [hunk 2]
```diff
@@ -35,20 +35,22 @@ The new password cannot already exist as a password for the user and must meet t
...
-    curl -k -v -X POST -H "content-type: application/json" -u "<administrator_user>:<password>" -d '{"username":"<username>", "old_password":"<an_existing_password>", "new_password":"<a_new_password>"}' https://<RS_server_address>:9443/v1/users/password
+    POST https://[host][:port]/v1/users/password
+    '{"username":"<username>", "old_password":"<an_existing_password>", "new_password":"<a_new_password>"}'
     ```
 
     After you run this command, you can authenticate with both the old and the new password.
 
 1. Update the password in all database connections that connect with the user account.
-1. Delete the original password with this DELETE command:
+1. Delete the original password with [`DELETE v1/users/password`]({{< relref "/rs/references/rest-api/requests/users/password#delete-password" >}}):
```
Comment (rrelledge) [inline content/rs/administering/access-control/password-rotation.md:48]:
```suggestion
1. Delete the original password with [`DELETE /v1/users/password`]({{< relref "/rs/references/rest-api/requests/users/password#delete-password" >}}):
```

File: content/rs/administering/troubleshooting/disabling-services.md [hunk 3]
```diff
@@ -9,9 +9,9 @@ The Redis Enterprise Software cluster nodes host a range of services that suppor
 In most deployments, either all of these services are required,
 or there are enough memory resources on the nodes for the database requirements.
 
-In a deployment with limited memory resources, certain services can be disabled from API endpoint to free system memory or using the `rladmin` command.
-Before you disable a service, make sure that your deployment does not depend on that service.
-After you disable a service, you can re-enable in the same way.
+In a deployment with limited memory resources, certain services can be turned off from API endpoint to free system memory or using the `rladmin` command.
+Before you turn off a service, make sure that your deployment does not depend on that service.
+After you turn off a service, you can re-enable in the same way.
```
Comment (rrelledge) [inline content/rs/administering/troubleshooting/disabling-services.md:14]:
```suggestion
In a deployment with limited memory resources, you can use the REST API or `rladmin` to turn off certain services to free system memory.
Before you turn off a service, make sure that your deployment does not depend on that service.
After you turn off a service, you can re-enable it in the same way.
```

File: content/rs/databases/import-export/flush-db-crdb.md [hunk 4]
```diff
@@ -98,22 +98,22 @@ To flush data from an Active-Active database:
...
-        curl -v -u <user>:<password> -X GET https://<cluster-fqdn>:9443/v1/crdbs
+        GET https://[host][:port]/v1/crdbs
         ```
 
-    1. To flush the Active-Active database, run:
+    1. To flush the Active-Active database, use [`PUT /v1/crdbs/{guid}/flush`]({{< relref "/rs/references/rest-api/requests/crdbs/flush#put-crdbs-flush" >}}):
 
         ```sh
-        curl -v -u <username>:<password> -X PUT https://<cluster-fqdn>:9443/v1/crdbs/<guid>/flush
+        PUT https://[host][:port]/v1/crdbs/guid/flush
```
Comment (rrelledge) [inline content/rs/databases/import-export/flush-db-crdb.md:110]:
```suggestion
        PUT https://[host][:port]/v1/crdbs/<guid>/flush
```

**Review by rrelledge — COMMENTED** [review]
Here's a few more suggestions.

**Comment by cmilesb** [comment]
Opened a new PR for this issue - #2004 - to bring in changes from the RS reorg.

**Pull request closed without merging**